python/examples/tabulated.py

```diff
@@ -0,0 +1,158 @@
+"""
+Tabulated radial integrals
+==========================
+
+.. start-body
+"""
+
+# This script illustrates how to generate splined radial basis functions/integrals,
+# using a "rectangular" Laplacian eigenstate (LE, https://doi.org/10.1063/5.0124363)
+# basis as the example, i.e, a LE basis truncated with l_max, n_max hyperparameters.
+
+
+import ase
+import numpy as np
+import scipy as sp
+from numpy import arange, pi, sqrt, zeros
+from scipy.optimize import brentq
+from scipy.special import jv
+from scipy.special import spherical_jn as j_l
+from scipy.integrate import quadrature
+
+from rascaline import SphericalExpansion, generate_splines
+
+
+# Spherical Bessel zeros (from the scipy cookbook)
+
+
+def Jn(r, n):
+    return sqrt(pi / (2 * r)) * jv(n + 0.5, r)
+
+
+def Jn_zeros(n, nt):
+    zerosj = zeros((n + 1, nt), dtype=np.float64)
+    zerosj[0] = arange(1, nt + 1) * pi
+    points = arange(1, nt + n + 1) * pi
+    racines = zeros(nt + n, dtype=np.float64)
+    for i in range(1, n + 1):
+        for j in range(nt + n - i):
+            foo = brentq(Jn, points[j], points[j + 1], (i,))
+            racines[j] = foo
```

Comment:
```suggestion
            racines[j] = brentq(Jn, points[j], points[j + 1], (i,))
```

is `racine` a word in english as well? This is the term for zero of a function in french =)

i.e. should this be `roots`?

Reply:
You're right. This is low-quality code from the scipy cookbook. It seems that although racine can be used as a synonym for root in all other contexts, that is not the case in math, so I'll change it